stetho/src/main/java/com/facebook/stetho/inspector/elements/android/TextViewDescriptor.java

```diff
@@ -32,6 +41,21 @@ protected void onUnhook(TextView element) {
     context.unhook();
   }
 
+  // TODO: Copied from ViewGroupDescriptor. Should we move it to the base class?
```

Comment:
I thought about doing that, but it ended up working better as a pattern ... for now anyway. Every descriptor seems to have slight variations on what it's doing with the ElementContext, and we'd have to do a bunch of clumsy stuff with generics and casting (I think). Over time we'll have a better idea as to whether this should be consolidated into the base Descriptor or ChainedDescriptor class.